presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/CachingHiveMetastore.java

```diff
@@ -156,7 +158,8 @@ public static CachingHiveMetastore memoizeMetastore(ExtendedHiveMetastore delega
                 maximumSize,
                 false,
                 ALL,
-                0.0);
+                0.0,
+                Optional.empty());
```

Comment:
what happens to memoize path... I'm not sure how this will impact the overall cache. But can we overload the cache stats so we have something like `NoopMetastoreCacheStats` instead of pass and checking optional everywhere in the downstream?